www/_posts/2021-08-21-removal-of-translated-docs.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Notice of the Removal of the Translated Docs"
+categories: announcements
+tags: news
+---
+
+We like to announce that we have deprecated and removed all translated docs.
+
+We had been providing translated docs since Cordova CLI 3.1.0 but, the number of resources has decreased to a point where it affects our ability to maintain them. The maintenance of these translated docs started to diverge around Cordova CLI 6.x.
+
+We have updated our webserver to redirect the language docs to their English counterparts. A 404 page will display only when the English counterparts do not exist. This issue occurs when translated docs continue to show deprecated content. For example, the BlackBerry 10 platform was deprecated and removed from Cordova CLI 8.x but, the translations were not updated and still show the deprecated platform.
+
+We like to thank the people who took the initiative in creating the translated docs and those who had put in their time and effort into maintaining them.
+
+For non-English speakers, we received feedback from various users that using services such as Google Translate to translate the English documentation into their language has produced better readable content over the existing translated docs. If you are using Google Chrome or Microsoft Edge, you could right-click and translate the page into your language.
```

Comment:
Nitpick: you can also do this in Safari. :)

Reply:
> If you are using Google Chrome or Microsoft Edge, you could right-click and translate the page into your language.

Safari did not fit well with this sentence about right-clicking. Safari does have some translation features but is limited to installed preferred languages, and not all are displayed. I test with English, Chinese & Japanese installed locally but, I could not translate the site into Japanese.

Reply:
You can right-click on the "translate" icon at the top. :) But yeah, fair, if Safari is an outlier in the number of languages, then it won't quite fit in.